modules/electrophysiology_browser/jsx/components/electrophysiology_session_panels.js

```diff
@@ -248,6 +251,7 @@ class FilePanel extends Component {
                   >
                     <button id='btn_download_all_files'
                             style={stylesFile.button.download}
+                            onClick={this.handleClick}
```

Comment:
handleClick is too vague maybe? I think something more descriptive could help (like annotationsUpdate).